tfhe/src/shortint/server_key/tests/shortint.rs

```diff
@@ -396,6 +401,49 @@ where
     assert_eq!(0, failures);
 }
 
+#[cfg(feature = "safe_serialization")]
+fn shortint_keyswitch_bootstrap_serialization_safe<P>(param: P)
```

Comment:
not sure what we are testing here after the serialization is done

Reply:
Removed